tests/test_client.py

```diff
@@ -98,6 +98,17 @@ def mock_e2b_api_key_none():
     tool_settings.e2b_api_key = original_api_key
 
 
+def test_initial_message_sequence(client: Union[LocalClient, RESTClient]):
+    messages = [{"role": "user", "text": "hello"}]
+    init_message_agent = client.create_agent(initial_message_sequence=messages)
```

Comment:
don't we already have a test for this somewhere? can we merge the two tests?

also can we have an additional test that checks for regression/degradation of the current default setup? we can also manually inspect by creating an agent in the CLI then doing `/dumpraw`